jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseBroadcaster.java

```diff
@@ -36,7 +36,7 @@
 public interface SseBroadcaster extends AutoCloseable {
 
     /**
-     * Register a listener, which will be called when an exception was thrown by a given SSE event output when trying to
+     * Register a listener, which will be called when an exception is thrown by a given SSE event output when tries to
```

Comment:
```suggestion
     * Register a listener, which will be called when an exception is thrown by a given {@link SseEventSink} when this SseBroadcaster tries to
```

"when tries" isn't clear who/what is trying.  My suggestion is to use "when this SseBroadcaster tries" instead.

Reply:
@andymc12  you're right it makes more sense like that.

Change done !

Thanks